### DISCUSSION TITLE
I am unable to download deDRM tools

### DISCUSSION THREAD

**Discussion by Clistgarten** [community/community#29215] [Other Feature Feedback, Questions, & Ideas]
I am unable to download deDRM tools to my Macbook Air mid 2013, OS Big Sur 11.6.8, Calibre 3.48. I have tried various Calibre & deDRM tools with the same result:invalid plugin.
![Screen Shot 2022-08-11 at 7 14 45 PM](https://user-images.githubusercontent.com/111097219/184263052-f2e9d4bb-6b7b-44e7-816c-e7f1617135d2.png)

**Comment by queenofcorgis** [comment]
Hi @Clistgarten,

Thanks for being a part of the GitHub Community! We're glad you're here.

I wanted to jump in and mention that if you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Calibre and deDRM. It's definitely possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
